docs/index.md

```diff
@@ -0,0 +1,48 @@
+# `lakefs-spec`: An `fsspec` backend for lakeFS
+
+Welcome to `lakefs-spec`, a [filesystem-spec](https://github.com/fsspec/filesystem_spec) backend implementation for the [lakeFS](https://lakefs.io/) data lake.
+Our primary goal is to streamline versioned data operations in lakeFS, enabling seamless integration with popular data science tools such as Pandas, Polars, and DuckDB directly from Python.
+
+Highlights:
+
+- High-level abstraction over basic lakeFS repository operations
+- Seamless integration into the `fsspec` ecosystem
+- Transaction support
+- Zero-config option through config autodiscovery
+- Automatic up-/download management to avoid unnecessary transfers for unchanged files
+
+<hr>
+
+::cards:: cols=3
+
+- title: Quickstart
+  content: Step-by-step installation and first operations
+  icon: ":octicons-flame-24:{ .landing-page-icon }"
+  url: quickstart.md
```

Comment:
I would terminate all of these with a full stop, just like the previous readme bullets.

Reply:
Then again, they're not full sentences.

Reply:
Okay, I'm good with that. But when I look at the screenshot again, the code appearance of `lakefs-spec` worsens the card's appearance by introducing a fourth color into the mix (violet for the emote, black for the text, white background).

Reply:
Fair point, I might do away with those

Reply:
Done in 6d2d0fb4482652061c96dc53bd67bd64b4e7ebc7